### PR TITLE
dev-python/python-gammu: enable py3.11

### PR DESCRIPTION
**Pull request by arkamar**
- enable py3.11
- convert to PEP517
- extend HOMEPAGE list

Closes: https://bugs.gentoo.org/896850

### CODE REVIEW

File: dev-python/python-gammu/python-gammu-3.2.4-r1.ebuild [hunk 1]
```diff
@@ -0,0 +1,52 @@
...
+RESTRICT="!test? ( test )"
+
+RDEPEND=">=app-mobilephone/gammu-1.34.0"
+DEPEND="${RDEPEND}"
+BDEPEND="
+	virtual/pkgconfig
+	test? (
+		$(python_gen_impl_dep sqlite)
+		app-mobilephone/gammu[dbi]
+	)"
```
Comment (mgorny) [inline dev-python/python-gammu/python-gammu-3.2.4-r1.ebuild:35]:
```suggestion
DEPEND="
	>=app-mobilephone/gammu-1.34.0
"
RDEPEND="
	${DEPEND}
"
BDEPEND="
	virtual/pkgconfig
	test? (
		$(python_gen_impl_dep sqlite)
		app-mobilephone/gammu[dbi]
	)
"
```

File: dev-python/python-gammu/python-gammu-3.2.4-r1.ebuild [hunk 2]
```diff
@@ -0,0 +1,52 @@
...
+
+DOCS=( AUTHORS NEWS.rst README.rst )
+
+distutils_enable_tests unittest
+
+python_test() {
+	rm -rf gammu
```
Comment (mgorny) [inline dev-python/python-gammu/python-gammu-3.2.4-r1.ebuild:42]:
```suggestion
	rm -rf gammu || die
```

**Review by mgorny — CHANGES_REQUESTED** [review]
Hmm…

> PEP 515 – Underscores in Numeric Literals

**Comment by arkamar** [comment]
Thanks for the review and suggested fixes. I have corrected PEP number in commit message.

**Comment by mgorny** [comment]
Thanks!

**Pull request closed without merging**